.eslintrc

```diff
@@ -8,7 +8,7 @@
 	},
 	"rules": {
 		"camelcase": 2,
-		"complexity": [2, 10],
+		"complexity": [2, 11],
```

Comment:
This means that it's time to refactor `ws.on('message')` callback, probably splitting it.

I usually have set a complexity limit of 7 or 8 as my goal :)